cinder/volume/api.py

```diff
@@ -84,13 +84,20 @@
                                help='Cache volume availability zones in '
                                     'memory for the provided duration in '
                                     'seconds')
+migrate_on_attach_opt = cfg.BoolOpt('allow_migration_on_attach',
+                                    default=False,
+                                    help="A host might recognise a connector"
+                                         "as valid but it can't use it to"
+                                         "initialize a connection. This will"
+                                         "allow to migrate the volume to a"
+                                         "valid host prior to attachment.")
```

Comment:
Need to add spaces between the end of the " and the beginning of the next line's "